### ISSUE TITLE
the number of queries in MSMARCO

### ISSUE THREAD

**Issue by mjeensung** [beir-cellar/beir#95]
Hi,

I have a question regarding the number of queries in MSMARCO.
According to the paper and the readme, the number of test queries in MSMARCO is 6,980.

However, when I ran the following codes, I was only able to get 43 queries.
```
>> corpus, queries, qrels = GenericDataLoader(data_folder='msmarco').load(split="test")
>> print(len(queries))
43
```

Instead, I got 6,980 queries from the dev set.
Should I use the dev queries when evaluating MSMARCO instead of the test queries?

Thanks!

**Comment by cadurosar** [comment]
Hi,

The split you are looking for is the "dev" split (so split="dev"). BEIR considers MSMARCO test to be one of the TREC-DL competitions.

**Comment by mjeensung** [comment]
@cadurosar 

Thanks! It's clear now.

**Comment by rahmanidashti** [comment]
Hi,

Thanks for question and the answer. Is there any documentation about the functions? For example, how `split` works? In MSMARCO, we have `9261` test samples, but the method returns `45`! What is the reason?

**Comment by YuchenHui22314** [comment]
Thanks for the clarification. However, this is very misleading, because for other BEIR datasets, we should use the "test" split. And suprisingly for MsMarco, we should use "dev". I used to use "test" split also for MsMarco and produced unexpected high accuracy. It took me a while to figure this out. So, it is ideal to mention this particularity in the homepage so as to save other researcher's time. @thakur-nandan
